atlasdb-cassandra-integration-tests/src/test/resources/docker-compose.yml

```diff
@@ -2,6 +2,6 @@ version: '2'
 
 services:
   cassandra:
-    image: com.palantir.docker/atlas-cassandra:2.2-v0.1
+    image: palantirtechnologies/docker-cassandra-atlasdb:latest
```

Comment:
That's surprising. We should definitely tag them next time we change the other repo, if not sooner. I wouldn't block on it for this PR.